bitnami/common/templates/_propertiesvalues.tpl

```diff
@@ -0,0 +1,5 @@
+{{- define "common.properties.render" -}}
+{{- range $key, $value := . }}
+{{ $key }}={{ $value }}
```

Comment:
Rendering the `$value` as a template would be useful I think

```suggestion
{{ $key }}={{ tpl $value $ }}
```

Reply:
fixed as proposed